release-engineering/role-handbooks/branch-manager.md

```diff
@@ -73,38 +72,45 @@ In this handbook, we will make several references to Kubernetes releases, milest
+| "previous release", "previous milestone" | `x.y-1` | Kubernetes 1.17 |
+| "next release", "next milestone" | `x.y+1` | Kubernetes 1.19 |
+| "release no longer in support" | `x.y-4` | Kubernetes 1.14 |
 
 **As an editor of this content, Branch Managers should periodically update these conventions and the examples contained within this handbook.**
 
+### Minimum Requirements
+
+- Familiarity with basic Unix commands and able to debug shell scripts.
+- Familiarity with branched source code workflows via `git` and associated `git` command line invocations.
+- General knowledge of Google Cloud (Cloud Build and Cloud Storage).
+- Open to seeking help and communicating clearly.
+- [Kubernetes Community Membership](https://github.com/kubernetes/community/blob/master/community-membership.md#member)
+
 ### Responsibilities
 
-The release branch manager is responsible for cutting a version of [Kubernetes]. Each release is a three month cycle where as branch manager:
-
-1. You will cut releases for the vX.Y cycle as specified on the Timeline in `sig-release/releases/release-x.y/README.md`.
-   - e.g. For cycle v1.16 you're expected to allocate time to cut releases on the dates detailed on [sig-release/releases/release-1.16/README.md][release-1.16].
-   The bulk of your time commitment is during release cut days.
-2. You must allocate time to setup your system to run the release tools; eventually you'll have a routine going and become more familiar with the process.
-3. Participate in weekly one hour release team meetings.
-4. Run `krel ff` on a daily basis as soon as the release branch i.e. `release-x.y` [is created](#branch-creation). (Less than 45mins)
-5. Your time commitment increases during code freeze (when approaching the official release), expect 3 or more 1 hour meetings in a week.
-6. Select Associates and guide them in preparation to become the section lead for the next cycle.
-7. Delegate tasks to the Associates (where applicable), so that they may exercise the release process.
-8. Update this handbook where appropriate for the next release cycle.
-9. Have willingness to accommodate with different timezone esp. for the release team and your Associates.
-10. Participate in conversations that happen on [#sig-release] and [#release-management]
+The release branch manager is responsible for cutting a version of [Kubernetes]. Each release is a three month cycle where as branch manager it's expected you:
+
+1. Cut releases for the next release cycle as specified on the timeline in `sig-release/releases/release-x.y/README.md`
+   - For example, during the v1.16 cycle you're expected to allocate time to cut releases on the dates detailed in [sig-release/releases/release-1.16/README.md][release-1.16]. The bulk of your time commitment is during release cut days.
+1. Allocate time to setup your system to run the release tools; eventually you'll become more familiar with the process.
```

Comment:
Maybe a mention here that this is ongoing.  Code change in the tooling drive change requirements in the host runtime system.  Reads as if it is a one-time up front cost.